File: CP website/WorkatTech/Arrays/Positive Cumulative Sum.java
// https://workat.tech/problem-solving/practice/positive-cumulative-sum
class Solution {
	List<Integer> getPositiveCumulativeSum (int[] arr) {
		// add your logic here
		List<Integer> l = new ArrayList<>();
		
		if(arr[0]>0){
				l.add(arr[0]);
			}
			
		for(int i=1;i<arr.length;i++){
			arr[i] = arr[i] + arr[i-1];
			if(arr[i]>0){
				l.add(arr[i]);
			}
			
		}
		return l;
	}
}

File: CP website/WorkatTech/Arrays/Positive Cumulative Sum.py
# https://workat.tech/problem-solving/practice/positive-cumulative-sum/
class Solution:
	def getPositiveCumulativeSum(self, arr: List[int]) -> List[int]:
		n = len(arr)
		pos = []
		if(arr[0] >0):pos.append(arr[0])
		for i in range(1,n):
			arr[i] = arr[i-1] + arr[i]
			if(arr[i]>0):pos.append(arr[i])
			
		return pos

